jwala-persistence/src/main/java/com/cerner/jwala/persistence/service/impl/JpaGroupPersistenceServiceImpl.java

```diff
@@ -22,23 +25,29 @@
 import javax.persistence.EntityManager;
 import javax.persistence.PersistenceContext;
 import javax.persistence.Query;
+import java.text.MessageFormat;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
 public class JpaGroupPersistenceServiceImpl implements GroupPersistenceService {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(JpaGroupPersistenceServiceImpl.class);
+
     private final GroupCrudService groupCrudService;
     private final GroupJvmRelationshipService groupJvmRelationshipService;
+    private ApplicationCrudService applicationCrudService;
 
 
     @PersistenceContext(unitName = "jwala-unit")
     protected EntityManager entityManager; // We're removing the CRUD layer in the near future so going forward, new methods will be using the entity manager in this class.
 
     public JpaGroupPersistenceServiceImpl(final GroupCrudService theGroupCrudService,
-                                          final GroupJvmRelationshipService theGroupJvmRelationshipService) {
+                                          final GroupJvmRelationshipService theGroupJvmRelationshipService,
+                                          ApplicationCrudService applicationCrudService) {
```

Comment:
Can you put final to applicationCrudService too ? Just so it's consistent with the other parameters.